README.md

```diff
@@ -13,11 +13,17 @@ It contains information about his projects, skills, experience and includes his
 - JAVASCRIPT
 
 
-
 ## Getting Started
 
-** To use this project, just clone it into your computer
-** open it with your presefered browser.
+## In order to have a copy of this project in your computer follow these steps:
+** Open your favourite git client
+** clone this project into your computer using this link https://github.com/Lasten-Ernest/Lasten-Ernest-Portfolio-Website.git 
+** open it with your prefered browser.
+
+
+## In order to access this website on line:
+** copy the link below and paste it into your browser's address bar.
+    https://lasten-ernest.github.io/
```

Comment:
- [ ] OPTIONAL: It is better to use links in github, so that users click on it and open the page, eg: [See the live preview](https://lasten-ernest.github.io/)
 As you can see it is less wordy and looks, here is how I made it: 
`[See the live preview](https://lasten-ernest.github.io/)`

Reply:
@ [ShoiraTa](https://github.com/ShoiraTa), Thanks for your comment, I will implement that.